### PR TITLE
Restructuring of build process

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
We should submit the docs to the plugin index before merging this PR.

**Review by appleboy — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -1,30 +1,96 @@
...
-  latest:
+  build_linux_amd64:
+    image: golang:1.9
+    group: build
+    environment:
+      - GOOS=linux
+      - GOARCH=amd64
+      - CGO_ENABLED=0
+    commands:
+      - go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -tags netgo -o release/linux/amd64/drone-slack-blame
```
Comment (tonglil) [inline .drone.yml]: You don't need `-tags netgo` to build static binaries anymore since Go 1.5.

**Comment by tonglil** [comment]
Otherwise LGTM.

File: .drone.yml (hunk 1 above)
Comment (tboerger) [reply .drone.yml]: This is a plain copy from the .drone.sh script provided by @bradrydzewski, but from my side I'm fine to update it like that

**Pull request merged**